serokell-style.md

```diff
@@ -109,12 +124,16 @@ data NetworkConfig = NetworkConfig
     }
 ```
 
+TODO: consider remove this rule? GHC has `-XOverloadedRecordFields` with
+[`HasField`](https://www.stackage.org/haddock/lts-10.4/base-4.10.1.0/GHC-Records.html#t:HasField)
+type class.
```

Comment:
I don't think we should remove this rule before we start using this stuff (`XOverloadedRecordFields` and `HasField`) very actively (which may happen after very long time or not happen at all).